### ISSUE TITLE
spacelift_policy renames require resource replacement

### ISSUE THREAD

**Issue by alexjurkiewicz** [spacelift-io/terraform-provider-spacelift#345]
```
  # spacelift_policy.base_plan_policy must be replaced
-/+ resource "spacelift_policy" "base_plan_policy" {
      ~ id       = "base-plan" -> (known after apply)
      ~ name     = "Base Plan" -> "base-plan" # forces replacement
      ~ space_id = "legacy" -> (known after apply)
        # (3 unchanged attributes hidden)
    }
```

Changing the name of a policy shouldn't force replacement. You can rename policies in-place in the Spacelift UI.

**Comment by marcinwyszynski** [comment]
On it.

**Comment by michieldewilde** [comment]
The issue has been addressed with the [v0.1.28](https://github.com/spacelift-io/terraform-provider-spacelift/releases/tag/v0.1.28) release.

**Comment by alexjurkiewicz** [comment]
Thank you!!!
